tests/test_lab_init.py

```diff
@@ -15,20 +15,6 @@
 
 
 class TestLabInit:
-    @patch(
-        "instructlab.config.init.convert_bytes_to_proper_mag", return_value=(89.0, "GB")
-    )
-    def test_ilab_config_init_auto_detection(
-        self, convert_bytes_to_proper_mag_mock, cli_runner: CliRunner
-    ):
-        result = cli_runner.invoke(lab.ilab, ["config", "init", "--interactive"])
-        assert result.exit_code == 0, result.stdout
-        convert_bytes_to_proper_mag_mock.assert_called_once()
-        assert (
-            "We chose Nvidia 2x A100 as your designated training profile. This is for systems with 160 GB of vRAM"
-            in result.stdout
-        )
```

Comment:
Will a new auto detection test be added as part of the auto detection PR?

Reply:
this test checks train profile auto detection specific functionality. It will be removed and then added in a follow up based off of https://github.com/cdoern/instructlab/tree/system-profiles-autodetection